packages/hardhat-core/test/fixture-projects/solidity-config-warnings/config-without-solidity.js

```diff
@@ -1 +1 @@
-module.exports = {};
+module.exports = { solidity: undefined };
```

Comment:
This feels a bit ugly. Maybe it would be better to add some other field to prevent the config from being empty, like:

```
module.exports = {
  networks: {}
}
```